moduleframework/pdc_data.py

```diff
@@ -44,10 +44,10 @@
     print_debug("ODCS  library cannot be imported. ODCS is not supported")
 
 
-PDC_SERVER = "https://pdc.fedoraproject.org/rest_api/v1/unreleasedvariants"
+PDC_SERVER = "https://pdc.fedoraproject.org/rest_api/v1/modules"
 ODCS_URL = "https://odcs.fedoraproject.org"
 DEFAULT_MODULE_STREAM = "master"
-BASE_REPO_URL = "https://kojipkgs.fedoraproject.org/compose/latest-Fedora-Modular-{}/compose/Server/{}/os"
+BASE_REPO_URL = "http://ftp.fi.muni.cz/pub/linux/fedora/linux/releases/{}/Workstation/{}/"
```

Comment:
This is really URL with Base module repo?

Reply:
modularity were changed much, now it is opt in, so we can use any repo. This original repo disappeared and all these composes were removed

Reply:
it's better to use mirror manager url and let it decide on closest mirror rather than hardcode:
```
https://download.fedoraproject.org/pub/fedora/linux/releases/
```

Reply:
makes sense. Thanks. I'll change it